### ISSUE TITLE
Improve performance

### ISSUE THREAD

**Issue by stat-yyang** [privefl/bigsnpr#183]
Hi, 

I thresholded the SNPs using the thresholding for Continuous trait. And I am using the LD structure computed from validation set. The performance of the summary statistics on test set is shown in the picture. The comparison with the other two methods (SBayesR and DBSLMM) shows that LDPred2 does not perform well (the trait in the graph is a continuous trait). Do you know how to further improve it?

Best
<img width="152" alt="Screen Shot 2021-02-17 at 10 34 00 AM" src="https://user-images.githubusercontent.com/47928843/108227868-52585900-710c-11eb-8238-fd3d298602ac.png">

**Comment by privefl** [comment]
I would need more details.

**Comment by privefl** [comment]
And what do you mean by thresholding the SNPs?

**Comment by stat-yyang** [comment]
Sure! We separate UKB samples into train, validation, and test set. Train set is used to generate summary statistics. Validation set is used to compute LD and tune the parameters. And then we store the beta from best model, apply beta to test set to obtain PRS. And see how PRS score helps improve R2. The plot shows the additional R2 of linear models when adding PRS.

Thresholding the SNPs are filtering out bad SNPs as indicating in the pipeline.

**Comment by privefl** [comment]
Is it simulations? Can you show the plot of predictive accuracy for all parameters?

Thresholding = p-value thresholding normally.

**Comment by privefl** [comment]
Any update on this?
